frontend/src/components/accounts/SignupSuccessMessage/index.jsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from 'react-router-dom';
+
+import styles from './SignupSuccess.module.css';
+
+
+function SignupSuccessMessage() {
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.success}>
+        <h4 className='h5'>Thank you for signing up</h4>
+        <p className={styles.instr}>
+          Checkout our <Link to='' className={styles.link}>home</Link> or{' '}
```

Comment:
to='/' instead of to='', this doesn't auto login the user, which is what I think you intend.

Reply:
It's doesn't auto login the user, I'm just trying to get it to go to the login page

Reply:
Fixed in [85a1a86](https://github.com/lujorryn/PetPal-G0346/pull/24/commits/85a1a86f83af31181ba802c724ed58323bc5967c)

It still doesn't auto-login the user though because it was never meant to do so.